blog/2015/04/how-to-contribute-with-template.html

```diff
@@ -0,0 +1,112 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
```

Comment:
I'd prefer to have people listed as authors - their identities tend to be more stable than those of committees.